docs/variants/pc_engines/hardware-matrix.md

```diff
@@ -0,0 +1,22 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the hardware configurations used for validation of the
+coreboot port on the PC Engines apu2 family of mainboards.
+
+=== "apu4"
+
+    | Component                      | Description                                      |
+    |--------------------------------|--------------------------------------------------|
+    | **CPU**                        | AMD GX-412TC SOC                                 |
+    |                                | Passive cooling to enclosure                     |
+    | **RAM**                        | 2 GB DDR3 non-ECC (soldered)                     |
+    | **SSD**                        | Apacer ST170 60GB                                |
+    | **Flash memory**               | Winbond W25Q64JV                                 |
+    | **USB pendrives**              | SanDisk Ultra USB 3.0 32 GB                      |
+    | **Wireless card**              | Compex WLE900VX                                  |
+    | **Network**                    | Local network wired connection                   |
```

Comment:
We typically included the network card model, didn't we?

Reply:
Added in 6011abe146c0